src/components/section/footer.jsx

```
@@ -4,8 +4,9 @@ import "../../assert/layout.css";
 
 const Footer = () => {
     return (
-        <footer id='footer' role="contentinfo"> 
+        <footer id='footer' role="contentinfo">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<footer>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)